### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by tomekw**

### CODE REVIEW

**Comment by ChuckJHardy** [comment]
There is nothing wrong with what you have done, yet what is the reason for the commit?

File: ltree_hierarchy.gemspec [hunk 1]
```diff
@@ -3,29 +3,29 @@ $:.push File.expand_path("../lib", __FILE__)
...
-  s.authors           = ["Rob Worley"]
-  s.email             = ["robert.worley@gmail.com"]
-  s.homepage          = "https://github.com/robworley/ltree_hierarchy"
-  s.summary           = "Organize ActiveRecord models into a tree using PostgreSQL's ltree datatype"
-  s.description       = "Organizes ActiveRecord models into a tree/hierarchy using a materialized path implementation based around PostgreSQL's ltree datatype. ltree's operators ensure that queries are fast and easily understood."
+  s.name = "ltree_hierarchy"
+  s.version = Ltree::Hierarchy::VERSION
+  s.authors = ["Rob Worley"]
+  s.email = ["robert.worley@gmail.com"]
+  s.homepage = "https://github.com/robworley/ltree_hierarchy"
```
Comment (cfabianski) [inline ltree_hierarchy.gemspec]: Might be worth changing this authors list and the contact info here as well now?

**Comment by tomekw** [comment]
@ChuckJHardy planning to fix at least one issue with `leaves` method, just a cleanup before doing real stuff.

File: ltree_hierarchy.gemspec (hunk 1 above)
Comment (tomekw) [inline ltree_hierarchy.gemspec]: @cfabianski 

``` ruby
s.authors = ["Leadformance"]`
s.email = ["dev@leadformace.com"]
```

?

Comment (cfabianski) [inline ltree_hierarchy.gemspec]:
```ruby
s.author = ["Rob Worley", "Leadformance"]
s.email = ["dev@leadformance.com"]
s.homepage = "https://github.com/Leadformance/ltree_hierarchy"
```

Comment (tomekw) [inline ltree_hierarchy.gemspec]: Done!

**Comment by cfabianski** [comment]
:star:

**Comment by ChuckJHardy** [comment]
:+1: :shipit:

**Pull request merged**